tasks/update-component-sbom/update-component-sbom.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: update-component-sbom
+  labels:
+    app.kubernetes.io/version: "0.0.1"
```

Comment:
No real way of knowing this, but we typically do 0.1.0 to start. Can you switch to this?